cmd/controller-manager/main.go

```diff
@@ -93,9 +95,9 @@ func main() {
 	// manager here is mainly used for leader election and health checks
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
 		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
+		Metrics:                crmetrics.Options{BindAddress: metricsAddr},
```

Comment:
Why is this part of this PR ?

Reply:
Because the old way was deprecated and is no longer available.

Reply:
We went through the same thing in https://github.com/kubestellar/ocm-status-addon/pull/76